### PR TITLE
Decompile unk_0203170C

### PR DESCRIPTION
**Pull request by abaresk**

### CODE REVIEW

File: src/unk_0203170C.c [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+    Unk0203170C *ptr2;
+    PLAYERPROFILE* profile;
+
+    ptr = sub_0202CA44(saveData);
+    profile = Sav2_PlayerData_GetProfileAddr(saveData);
+    ptr2 = sub_02031774(saveData);
+
+    MI_CpuFill8(a1, 0, sizeof(Unk020317F4));
+    a1->unk0 = GAME_VERSION;
+    a1->unk1 = 2;
```
Comment (tgsm) [inline src/unk_0203170C.c:84]:
```suggestion
    a1->unk1 = GAME_LANGUAGE;
```

File: include/unk_0203170C.h [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    u8 unk33;
+    u32 unk34;
+    u16 unk38;
+    u16 unk3a;
+    u32 unk3c;
+} Unk0203170C;
+
+typedef struct {
+    u8 unk0;
+    u8 unk1;
```
Comment (tgsm) [inline include/unk_0203170C.h:15]:
```suggestion
    u8 version;
    u8 language;
```

File: include/unk_0203170C.h [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    u8 unk2;
+    u8 unk3;
+    u32 unk4;
+    u16 unk8[OT_NAME_LENGTH + 1];
```
Comment (tgsm) [inline include/unk_0203170C.h:19]:
```suggestion
    u32 otid;
    u16 name[OT_NAME_LENGTH + 1];
```

**Review by tgsm — CHANGES_REQUESTED** [review]

**Comment by abaresk** [comment]
Updated field names in structs

**Review by tgsm — DISMISSED** [review]

**Review by PikalaxALT — CHANGES_REQUESTED** [review]

**Comment by abaresk** [comment]
Fixed

**Review by PikalaxALT — APPROVED** [review]

**Pull request merged**